### ISSUE TITLE
Amount of training steps

### ISSUE THREAD

**Issue by MartimQS** [lucidrains/lightweight-gan#66]
If I bring down the number of training steps from 150 000 to 30 000, will the trained model be overall bad? Does it really need the 100 000 or 150 000 training steps?

**Comment by Dok11** [comment]
I have been 550 000+ steps for my images and result still not enough good.
But in fact the result was not improve (subjective, I didnt calc FID) after 30-60k steps.

**Comment by MartimQS** [comment]
Have you tried logging the values for the loss functions and gp in a file and visualizing the results?

**Comment by Dok11** [comment]
The loss values is not informative and dont have relative to visual quality like FID, but FID require other installed library and a lot of time so I still not calulate this =)
Visualising result is enabled by default, and I made command to show progress (see readme.md) and I really dont see improve after 60k steps (with batch size equal to 3)

**Comment by MartimQS** [comment]
The visualization of the result shows the generated images. However, I’m still improving the training phase and discovering the best settings combination for training, that is why I wanted to know the behavior of the networks with each iteration
